packages/flutter/lib/src/material/tabs.dart

```diff
@@ -120,7 +124,11 @@ class _TabStyle extends AnimatedWidget {
   @override
   Widget build(BuildContext context) {
     final ThemeData themeData = Theme.of(context);
-    final TextStyle textStyle = themeData.primaryTextTheme.body2;
+    final TextStyle defaultStyle = labelStyle ?? unselectedLabelStyle ?? themeData.primaryTextTheme.body2;
```

Comment:
It's a little strange for the selected state to fall back to the `unselectedLabelStyle`.  I'd expect `labelStyle` to affect both states if I didn't give an `unselectedLabelStyle`, but I wouldn't expect `unselectedLabelStyle` alone to affect the selected state.